aspnetcore/blazor/fundamentals/startup.md

```diff
@@ -72,6 +90,94 @@ For an examples of JS initializers, see the following resources:
 
 [!INCLUDE[](~/includes/aspnetcore-repo-ref-source-links.md)]
 
+### Ensure libraries are loaded in a specific order
+
+Append custom scripts to the `<head>` in `beforeStart` and `afterStarted` in the order that they should load.
+
+The following example loads `script1.js` before `script2.js` and `script3.js` before `script4.js`:
+
+```javascript
+export function beforeStart(options, extensions) {
+    var customScript1 = document.createElement('script');
+    customScript1.setAttribute('src', 'script1.js');
+    document.head.appendChild(customScript1);
+
+    var customScript2 = document.createElement('script');
+    customScript2.setAttribute('src', 'script2.js');
+    document.head.appendChild(customScript2);
+}
+
+export function afterStarted(blazor) {
+    var customScript1 = document.createElement('script');
+    customScript1.setAttribute('src', 'script3.js');
+    document.head.appendChild(customScript1);
+
+    var customScript2 = document.createElement('script');
+    customScript2.setAttribute('src', 'script4.js');
+    document.head.appendChild(customScript2);
+}
+```
+
+### Import additional modules
+
+Use top-level [`import`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Statements/import) statements in the JS initializers file (`*.lib.module.js`) to import additional modules.
+
+`additionalModule.js`:
+
+```javascript
+export function logMessage() {
+  console.log('logMessage is logging');
+}
+```
+
+```javascript
+import { logMessage } from "/additionalModule.js";
+
+export function beforeStart(options, extensions) {
+  ...
+
+  logMessage();
+}
+```
+
+For browser compatibility, see [Can I use: JavaScript statement: import](https://caniuse.com/?search=JavaScript%20statement%3A%20import).
+
+[Dynamic import with the `import()` operator](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/import) is supported with ASP.NET Core and Blazor:
+
+```javascript
+import("/additionalModule.js");
```

Comment:
I'm not sure we need an example showing dynamic imports. I can't really think of a real case where you'd want to use dynamic imports over import statements in the context of a JS initializer. Blazor dynamically imports your JS initializer just before calling its `beforeStart()` function, so using an import statement and doing a dynamic import in `beforeStart()` effectively do the same thing.

Reply:
I'll strike it from this section/article.

Is the dynamic import useful for loading modules based on a conditional check? If so, we can keep it with the rest of the 'where to place JS in a Blazor app (modules)' content (which isn't in the this article). If yes, I'll add it to the right spot. If not, I'll strike it here, and we won't keep any coverage on it.

Reply:
> Is the dynamic import useful for loading modules based on a conditional check?

Yep, that does seem reasonable! I suppose it would be useful to mention 🙂